polygon/bridge/reader.go

```diff
@@ -46,25 +47,22 @@ func NewReader(store Store, log log.Logger, stateReceiverContractAddress string)
 func (r *Reader) Events(ctx context.Context, blockNum uint64) ([]*types.Message, error) {
 	start, end, err := r.store.EventIDRange(ctx, blockNum)
 	if err != nil {
-		return nil, err
-	}
-
-	if end == 0 { // exception for tip processing
-		end, err = r.store.LastProcessedEventID(ctx)
-		if err != nil {
-			return nil, err
+		if errors.Is(err, ErrEventIDRangeNotFound) {
+			return nil, nil
 		}
+
+		return nil, err
 	}
 
 	eventsRaw := make([]*types.Message, 0, end-start+1)
 
 	// get events from DB
-	events, err := r.store.Events(ctx, start+1, end+1)
+	events, err := r.store.Events(ctx, start, end+1)
```

Comment:
What's the reason for this change?

Reply:
`EventIDRange` now returns [start, end] instead of (start, end]. Events is [start, end).